skimmer/skimmer_utils.py

```diff
@@ -3,11 +3,13 @@
 import numba as nb
 from coffea.nanoevents.methods import vector
 import uproot
+import pickle  
 
 from utils.awkward_array_utilities import as_type
 from utils.tree_maker.triggers import trigger_table as trigger_table_treemaker
 from utils.systematics import calc_jec_variation, calc_jer_variation
 from utils.Logger import *
+from analysis_configs import objects_definition as obj
```

Comment:
The logic of the code should be rethought here: this script is meant to have functions to help defining the selections, it should not contain special definitions for any user. Here the code will only work for us and not Cesare or any students we have in the group. What about passing to the `apply_hem_veto` function directly the good jets, electrons, and muons?